owa/app/css/openmrs-owa-idgen.css

```diff
@@ -117,21 +118,232 @@ nav{
     padding: 0 !important;
 }
 
-
-.navbar-light .navbar-nav .nav-link{
+.navbar-light .navbar-nav .nav-link {
     color: #eeeeee !important;
```

Comment:
Ditto. Please find all the instances you used `!important` and fix them.

Reply:
@kingisaac95 I will sync with you tommorow so that you show me how, most of the ! important were part of my PR